bgpd/bgp_evpn_vty.c

```diff
@@ -1011,10 +1027,13 @@ static void show_vni_entry(struct hash_bucket *bucket, void *args[])
 	char *ecom_str;
 	struct listnode *node, *nnode;
 	struct ecommunity *ecom;
+	struct bgp *bgp_evpn;
 
 	vty = args[0];
 	json = args[1];
 
+	bgp_evpn = bgp_get_evpn();
```

Comment:
shouldn't we check if bgp_evpn is valid after the assignment?

Reply:
The return only allows/prevents printing some things... I think "invalid" would result in the same outcome as "not existing," correct? In this case, I'm not certain it needs to be checked (?).

Reply:
bgp_evpn is always valid for bgp default instance, so the check is not required.